daos/LikeDao.ts

```diff
@@ -5,6 +5,8 @@
 import Like from "../models/Like";
 import LikeModel from "../mongoose/LikeModel";
 import LikeDaoI from "../interfaces/LikeDao";
+import Bookmark from "../models/Bookmark";
+import BookmarkModel from "../mongoose/BookmarkModel";
```

Comment:
no change needed

Reply:
I don't see any significant change in the code. I don't think this pull request will add anything to the implemented like service. Morever, the like service is tested already. Please, let me know if you still think it is necessary to push this PR.

Reply:
You can't merge thr PR in main. You have to merge in spring5-integration-node

Reply:
Please close this PR and make a new one with valid code changes.